### PR TITLE
live555: Update to version 2019.05.03

### PR DESCRIPTION
**Pull request by stenstorp**
Version 2019.02.03 no longer available. Update to 2019.05.03.

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -2841,7 +2841,7 @@ libyaz_icu.so.5 yaz-5.23.1_1
 libyaz.so.5 yaz-5.23.1_1
 libyaz_server.so.5 yaz-5.23.1_1
 libgcab-1.0.so.0 gcab-0.7_1
-libliveMedia.so.65 live555-2018.12.14_1
+libliveMedia.so.66 live555-2018.12.14_1
```
Comment (Hoshpak) [inline common/shlibs]: According to `xrevshlib`, `vlc` depends on this library so it needs to be rebuilt on soname bump.

Comment (stenstorp) [reply common/shlibs]: So bump the revision on VLC too?

**Comment by Hoshpak** [comment]
Yes please.

Am 3. Mai 2019 23:49:31 MESZ schrieb stenstorp <notifications@github.com>:

**Comment by Hoshpak** [comment]
Oh, I didn't explicitely mention it but the vlc revbump should be a separate commit. Something along the lines of `vlc: rebuild against live555-2019.05.03`. And the common/shlibs line should mention the version that started providing the soname (presumably 2019.05.03) and not the old one.

**Pull request merged**